Tests/BasicsTests/ConcurrencyHelpersTests.swift

```diff
@@ -42,17 +44,18 @@ final class ConcurrencyHelpersTest: XCTestCase {
             }
 
             switch sync.wait(timeout: .now() + .seconds(2)) {
```

Comment:
Consider replacing this switch with just:

```swift
try #require(sync.wait(...) == .success)
expected.forEach { ... }
```

Reply:
DONE